kubernetes/dashboard_chart/templates/liqodash_deployment.yaml

```diff
@@ -0,0 +1,440 @@
+apiVersion: v1
```

Comment:
2 cents: I think this file should be splitted in few ones, it's a bit too long to be easily reviewed.